kyuubi-server/src/main/scala/org/apache/kyuubi/events/KyuubiSessionEvent.scala

```diff
@@ -23,7 +23,7 @@ import org.apache.kyuubi.server.KyuubiServer
 import org.apache.kyuubi.session.KyuubiSessionImpl
 
 /**
- * @param sessionId server session id
```

Comment:
we can keep the name and name `engineSessionId` as `remoteSessionId` just like other events